addons/web/static/src/views/fields/many2many_binary/many2many_binary_field.scss

```diff
@@ -42,7 +42,7 @@ $o-attachment-margin: 5px;
         padding-right: 40px;
     }
 
-    .o_image {
+    .o_preview_image {
```

Comment:
why that change ? I guess it's to not have `o_image` css properties on the `<img>` tag, but now if you upload a broken image (eg a text file renamed ".png"), you get
![image](https://github.com/user-attachments/assets/7462ffcd-639e-4993-b534-897e119085f1)

And without this change, you get
![image](https://github.com/user-attachments/assets/7005c655-55e3-4a74-ba2c-bbde566d0459)

(because you have stuff in `web/static/src/scss/mimetypes.scss` for that)

Reply:
Hello @std-odoo :slightly_smiling_face: 
If I keep the `o_image` class on our preview image we have this kind of ugly display:
![image](https://github.com/user-attachments/assets/366d6cbb-e29a-4b4c-97ea-21291c7deb53)
(for the odoo.png, the `o_image` class is set but without data-mimetype on our preview so it displays the unknown background behind)

What about keeping my change and add this attribute `onerror` on the preview?
```xml
<img t-if="isImage(file)" t-attf-src="/web/image/{{ file.id }}" onerror="this.src = '/web/static/img/mimetypes/unknown.svg'" class="o_preview_image o_hover object-fit-cover rounded"/>
```
Which will result in this display:
![image](https://github.com/user-attachments/assets/a3c5fba0-cf3c-451e-9459-28de08fabae9)

Reply:
Hi @amdi-odoo 🙂

Ah indeed, didn't see because I used image with a background and broken image looks different on Firefox 😅 

Can we just use `/web/static/img/mimetypes/image.svg` instead of "unknown" ?
(my example was bad sorry, because the mimetype attribute was not on the `<img>` and so it took `unknown`)